### PR TITLE
rename plan -> render_plan

### PR DESCRIPTION
**Pull request by grosser**
@zdrve

### CODE REVIEW

File: test/kennel/syncer_test.rb [hunk 1]
```diff
@@ -129,12 +129,11 @@ def change(*args)
 
   capture_all # TODO: pass an IO to syncer so we don't have to capture all output
 
-  describe "#plan" do
+  describe "#render_plan" do
     let(:plan) do
       (monitors + dashboards).each { |m| m[:id] ||= 123 } # existing components always have an id
-      syncer.plan
+      syncer.render_plan
```
[the remark this answers is not in the document]
Comment (grosser) [reply test/kennel/syncer_test.rb:135]: yeah that might be useful

[the remark this answers is not in the document]
Comment (grosser) [reply test/kennel/syncer_test.rb:135]: done 👁️

**Review by zdrve — APPROVED** [review]
Might be nice to test `plan` and `print_plan` separately but we can do that another day if we have nothing better to do ;-)

**Comment by grosser** [comment]
yeah, would mean lots of re-work in the tests for no real gain ...

**Pull request merged**